package.json

```diff
@@ -14,20 +14,20 @@
 		"generate-release": "node ./scripts/generateRelease.js",
 		"start": "webpack --env.development --env.watch",
 		"zip": "gulp zip",
-		"prettier": "prettier '**/*.{js,jsx,ts,tsx}'",
+		"prettier": "prettier \"**/*.{js,jsx,ts,tsx}\"",
```

Comment:
Still not able to match anything on windows of some strange reason. (husky / lint-staged still works though)

Reply:
Sorry I was mistaken. Now it works!